Wireframe/index.html

```diff
@@ -10,24 +10,92 @@
+            <br>
+            <br>
+            In summary, branches in Git allow developers to work on different tasks in isolation, organize development, and merge changes in an efficient and controlled way. They help manage multiple versions of a project simultaneously, making it easier to collaborate and experiment.
+            <br>
+            <br>
+          </p>
+             
+          <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches" target="_blank">Read more
+
+          </a>
+        </article>
+      </div>
+
+      <div class="bottom-side">
+        <article>
+          <img src="placeholder.svg" alt="" />
+          <h2>Readme file</h2>
+          <p>
+            A README file in Git (usually named README.md) is a documentation file that provides essential information about a project. It’s typically the first thing a person sees when they visit a repository and serves as a guide to understand what the project is, how to use it, and how to contribute to it.
+            <br>
+            <br>
+            You can add a README file to a repository to communicate important information about your project. A README, along with a repository license, citation file, contribution guidelines, and a code of conduct, communicates expectations for your project and helps you manage contributions.
+            <br>
+            <br>
+            Why a README is Important:
+            <br>
+            <br>
+            <strong>Clarifies Purpose:</strong>It explains what the project does, helping users quickly understand its purpose.
+            <br>
+            <strong>Guides Users:</strong> It provides instructions on installation, usage, and configuration.
+            <br>
+            <strong>Helps Contributions:</strong> It gives clear guidelines for people who want to contribute to the project.
+            <br>
+            <strong>Improves Accessibility:</strong> It makes the project more approachable and easier to use for both developers and non-developers.
+            <br>
+            <br>
+            In summary, a README file in Git is essential for documenting a project. It helps developers and other users understand the project’s purpose, how to use it, and how to contribute. It's a key part of every repository, especially for open-source projects.
+          </p>
+          <a href="https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#about-readmes">Read more</a>
+        </article>
+      </div>
+    </section>
     <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+      <div>
+        <p>
+          Code your future 2024.
+        </p>
+      </div>
```

Comment:
Footer position is fixed but you should also show footer's background, if not user can't clearly see where footer starts and also due to z-index problem, it creates  barrier for user to click on read more button if you scroll to top, please refers to these issue.